app/helpers/participants_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ParticipantsHelper
   def hall_of_fame_title
-    params[:action] == "hall_of_fame" ? "Hall Of Fame" : "Organizer Of Fame"
+    params[:action] == "hall_of_fame" ? "HALL OF FAME_" : "ORGANIZER OF FAME_"
```

Comment:
@frmendes Sure, makes sense after merging #80 . Can't really come up with a Portuguese translation for `Hall Of Fame`... `Corredor Da Fama`, `Passeio da Fama`? Any Ideas? 🤔